ttnn/cpp/ttnn/operations/data_movement/tilize_with_val_padding/tilize_with_val_padding_common.hpp

```diff
@@ -0,0 +1,9 @@
+// SPDX-FileCopyrightText: © 2024 Tenstorrent Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+namespace ttnn {
```

Comment:
I got confused since we now have tt_metal::Padding::PadValue and ttnn::PadValue
first one is enum and referes to type of padding (inf, zero, any) and this one is just a union of uint32 and a float